tests/pos/i14083.scala

```diff
@@ -0,0 +1,3 @@
+class Outer {
+  class Inner
+}
```

Comment:
You can turn this into a run test with:
```suggestion
@main def Test =
  assert(classOf[Outer#Inner].getConstructors.head.getParameters.head.isSynthetic)
```

Reply:
Ok. And we also want to check that the field is synthetic.